app/src/features/projects/view/components/__snapshots__/GeneralInformation.test.tsx.snap

```diff
@@ -138,9 +138,7 @@ exports[`GeneralInformation renders correctly 1`] = `
         >
           <h6
             class="MuiTypography-root MuiTypography-subtitle1"
-          >
-            Placeholder
-          </h6>
```

Comment:
same here, seems like `projectWithDetailsData.climate_change_initiatives` needs to be mocked or set so that we can see a placeholder being populated correctly in these snaps

Reply:
Fixed!